Formula/python@3.9.rb

```diff
@@ -75,6 +75,16 @@ class PythonAT39 < Formula
     sha256 "99a22d87add3f634ff917310a3d87e499f19e663413a52eb9232c447aa646c9f"
   end
 
+  # Remove this block when upstream adds arm64 and Big Sur compatibility
+  if MacOS.version >= :big_sur
```

Comment:
Isn't it more arch dependent then os?

Reply:
It's needed for both arm64 and Intel big sur